presto-main/src/test/java/com/facebook/presto/operator/TestAggregationOperator.java

```diff
@@ -105,7 +105,11 @@ public void testAggregation()
                         LONG_SUM.bind(ImmutableList.of(3), Optional.empty()),
                         REAL_SUM.bind(ImmutableList.of(4), Optional.empty()),
                         DOUBLE_SUM.bind(ImmutableList.of(5), Optional.empty()),
-                        maxVarcharColumn.bind(ImmutableList.of(6), Optional.empty())));
+                        maxVarcharColumn.bind(ImmutableList.of(6), Optional.empty())),
```

Comment:
please add tests to `AbstractTestQueries` with various cases:
* order by aggregations and rollup
* order by aggregations in `ORDER BY` clause
* order by aggregations with different ordering
* etc..
@pnowojski more ideas?